spring-cloud-alibaba-examples/integrated-example/integrated-account/pom.xml

```diff
@@ -24,7 +24,7 @@
 		<dependency>
 			<groupId>mysql</groupId>
 			<artifactId>mysql-connector-java</artifactId>
-			<version>5.1.47</version>
```

Comment:
Can the version tag be removed? I think using the version of Spring management is better.